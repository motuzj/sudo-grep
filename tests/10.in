recursive word search
-iR "a E" tests/texts
