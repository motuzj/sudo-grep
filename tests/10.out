tests/texts/text2.txt:elit non euismoda erat
tests/texts/text2.txt:Erosa erat nam porta rutrum
tests/texts/text4.txt:Chtenff a erat r'luh ngnw Azathothnth
tests/texts/text3.txt:Amatomaa erat nam borrar el cer
tests/texts/text1.txt:Non consectetur a erat nam at lectus urna.
